platform/windows/detect.py

```diff
@@ -195,7 +195,9 @@ def configure_msvc(env, manual_msvc_config):
 
     ## Compile/link flags
 
-    env.AppendUnique(CCFLAGS=['/MT', '/Gd', '/GR', '/nologo', '/utf-8'])
+    env.AppendUnique(CCFLAGS=['/MT', '/Gd', '/GR', '/nologo'])
+    if int(env['MSVC_VERSION'].split('.')[0]) >= 14: #vs2015 and later
```

Comment:
I think we could just use string/alphabetic comparison here like `if env['MSVC_VERSION'] >= "14":`

Reply:
I am fine with either. Both will work while making different assumptions (“begin with digits + .” vs “begin with two digits”). To me comparing number is a bit more intuitive than comparing strings. So I think it comes down to what the repository owner prefers.

Reply:
IMHO shorter string version is easier to read. Other than that, number version is good for me.

Reply:
I think I’ll keep this for now until there is a third vote. Thanks!